integration/auth/user_mgt_test.go

```diff
@@ -334,6 +587,124 @@ func TestDeleteUser(t *testing.T) {
 	}
 }
 
+func TestDeleteUsers(t *testing.T) {
+	// Creates a user and returns its uid. Upon failure, triggers t.Fatalf().
+	createTestUserOrDie := func(t *testing.T) string {
```

Comment:
Use the existing `newUserWithParams()` helper.

Reply:
Done.